### DISCUSSION TITLE
Call for proposals

### DISCUSSION THREAD

**Discussion by dhh1128** [trustoverip/trust-spanning-protocol#10] [General]
After discussing with other TF chairs, this discussion is a formal suggestion that the TF enter into a new phase where we entertain proposals (or perhaps, articulations of a coherent vision) about what the TSP consists of. These would not be simply a reiteration of the high-level requirements that have already been captured, but rather an answer to the question, "What, specifically, would you put into the TSP from any of the forerunner protocols and/or from other sources, and what would you change or leave out of scope from them?"

Doing this will allow us to see how closely aligned we already are (or not).

I suggest that we dedicate the next few TF meetings to hearing and discussing these proposals, and try to get 3-5 of them out on the table so we can compare and contrast. I'm imagining proposals that each consist of 30-40 minutes of presented material plus a moderate amount of live Q&A.

Per @talltree's wisdom, I further suggest that we start up a discussion about each of these proposals, so we can do Q&A about each in an asynchronous fashion.

I believe @SmithSamuelM wants to make a proposal like this at next Wednesday's meeting. I would also like to offer one at another TF meeting sometime soon. I think @wenjing also has thoughts he'd like to share.

Who else wants to sign up to present a proposal of this kind?

**Comment by 2byrds** [comment]
Looking forward to this new phase! The timing seems good as the introductory phase seems to be winding down.

**Comment by SmithSamuelM** [comment]
Also as a precursor to my presentation this week, watch the presentation I gave in the APAC session of the TSWG meeting on wednesday. It was on the Hourglass principle and include detailed diagrams of how routring and discovery fits into the model.

**Reply by talltree** [reply]
Ha! Sam, you just beat me to the punch. Here's what I just posted in the ["forerunners" discussion thread](https://github.com/trustoverip/trust-spanning-protocol/discussions/5):

I have a second very strong recommendation for all TSPTF members: take a hour in the next few days (ideally before our next TSPTF meetings this coming Wednesday) to watch [this presentation @SmithSamuelM made about the Hourglass Model](https://zoom.us/rec/share/rddaWi5PcDp_axmoLhK617WbmM3EWpFUBfX2GS4q_feuhhzn_Of9Yy0cSuS3IFFw.8DLf29JpSmCViVc0) at last Wednesday's APAC meeting of the Trust Registry TF.

Sam uses a series of diagrams to walk through the history of the Hourglass Model and show precisely how the IP spanning layer works in the TCP/IP stack — with a special emphasis on how the IP layer is — and **is not** — involved with discovery and routing.

Sam gave that talk to set the stage for the TSP proposal he will be presenting in this Wednesday's (Feb 8) TSPTF meetings. So it will be extremely helpful to anyone attending that meeting to have watched Sam's talk first.

_(Note: I will be cross-posting this message in GitHub, Slack, and email to make sure everyone sees it.)_

**Comment by talltree** [comment]
Per @dhh1128's post opening this discussion, we are planning to begin the Proposal stage of this task force (see [a complete list of the 10 stages](https://github.com/trustoverip/trust-spanning-protocol/discussions/6#discussioncomment-4876863)) starting with our next meetings beginning this coming Wednesday (Feb 8). **Note that this was originally scheduled to be a Technology Architecture TF meeting.** However at last week's Technology Stack WG Plenary meeting, we decided that the TATF can do its work asynchronously for the time being, as we need more time for the TSPTF work. So now TSPTF meetings will be 3 out of every 4 Wednesdays (the fourth will be the TSWG Plenary meeting).

The first proposal will be from @SmithSamuelM. As Daniel said, the format will be ~40 mins of presentation and ~20 mins of Q&A about that specific proposal. Once again, I strongly encourage everyone to watch [this 60 min recording of Sam's presentation on the Hourglass Model](https://zoom.us/rec/share/rddaWi5PcDp_axmoLhK617WbmM3EWpFUBfX2GS4q_feuhhzn_Of9Yy0cSuS3IFFw.8DLf29JpSmCViVc0) before next Wednesday's meeting as it lays the entire groundwork for his proposal.

Sam will present in the NA/EU meeting, and then if necessary repeat it in the APAC meeting. However we will share the NA/EU recording as soon as we can so that folks attending the APAC meeting will — if they have time — be able to watch it in advance, giving up more time for discussion in the APAC meeting.

On the two following Wednesdays, we will have proposals from @dhh1128 and Wenjing in the same format.

Anyone else who would like to schedule time for a proposal, just post here in this Discussion thread.
